eos/constraints/Bs-to-phi-ll.yaml

```diff
@@ -10,7 +10,7 @@ B_s->phimu^+mu^-::NormalizedBR[0.10,2.00]@LHCb:2015C:
     type: Gaussian
     observable: B_s->phill::NormalizedBR
     kinematics: {q2_min: 0.1, q2_max: 2}
-    options: {l: mu, psi: J/psi, tag: GvDV2020}
+    options: {l: mu, psi: J/psi, tag: GvDV2020, q: s}
```

Comment:
Hm, isn't this option better forced by the ``B_s->phill::NormalizedBR`` observable?

Reply:
Definitely!